packer/scripts/create-kube-certs.sh

```diff
@@ -2,6 +2,8 @@
 
 set -o errexit -o xtrace
 
+export PATH="${PATH}:/usr/local/bin/"
```

Comment:
Why don't you run the above script so we can expect `/usr/local/bin` in PATH then?